test/client/index.js

```diff
@@ -546,6 +546,51 @@ module.exports = config => {
     return this.api.consumeRecoveryCode(this.sessionToken, code, options)
   }
 
+  Client.prototype.createRecoveryKey = function (recoveryKeyId, recoveryData) {
+    return this.api.createRecoveryKey(this.sessionToken, recoveryKeyId, recoveryData)
+  }
+
+  Client.prototype.getRecoveryKey = function (recoveryKeyId) {
+    if (! this.accountResetToken) {
+      throw new Error('call verifyPasswordResetCode before calling resetPassword')
+    }
+
+    return this.api.getRecoveryKey(this.accountResetToken, recoveryKeyId)
+  }
+
+  Client.prototype.resetAccountWithRecoveryKey = function (newPassword, kB, recoveryKeyId, headers, options) {
+    if (! this.accountResetToken) {
+      throw new Error('call verifyPasswordResetCode before calling resetPassword')
+    }
+
+    var email = this.email
+    if (options && options.emailToHashWith) {
+      email = options.emailToHashWith
+    }
+
+    return this.setupCredentials(email, newPassword)
+      .then(function (/* bundle */) {
+        const wrapKb = butil.xorBuffers(kB, this.unwrapBKey)
+
+        return this.api.accountResetWithRecoveryKey(
+          this.accountResetToken,
+          this.authPW,
+          wrapKb,
+          recoveryKeyId,
+          headers,
+          options
+        )
+          .then(response => {
+            // Update to the new verified tokens
+            this.sessionToken = response.sessionToken
+            this.keyFetchToken = response.keyFetchToken
```

Comment:
Is it OK to replace `keyFetchToken` here even if you didn't request keys, or should this be guarded by an `if (options.keys)`?